O/ONNXRuntime/ONNXRuntime_CUDA/build_tarballs.jl

```diff
@@ -7,34 +7,36 @@ include(joinpath(YGGDRASIL_DIR, "fancy_toys.jl"))
 include(joinpath(YGGDRASIL_DIR, "platforms", "cuda.jl"))
 
 name = "ONNXRuntime_CUDA"
-version = v"1.10.0"
+version = v"1.19.0"
 
 # Cf. https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements
 # Cf. https://onnxruntime.ai/docs/execution-providers/TensorRT-ExecutionProvider.html#requirements
+# Cf. https://github.com/microsoft/onnxruntime/blob/v1.19.0/tools/ci_build/github/azure-pipelines/stages/py-cuda-packaging-stage.yml#L35
 cuda_versions = [
-    v"10.2",
-    v"11.3", # Using 11.3, and not 11.4, to be compatible with CUDNN v8.2.4, and with TensorRT (JLL) v8.0.1 (where the latter includes aarch64 support).
+    v"11",
+    v"12",
```

Comment:
There is also v13 by now.

But is this even used anywhere?

You are using `CUDA.supported_platforms` which I think brings its own list of CUDA versions?